src/styles/blocks/nav.scss

```
@@ -0,0 +1,43 @@
+.nav__ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  margin: 0;
+}
+
+.nav__item {
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-color;
+  line-height: 60px;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+
+  &:hover {
+    color: $secondary-color;
+    transition: all 0.3s ease;
```

don't use it like that, because it's the same as transition: 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```